.travis.yml

```diff
@@ -12,5 +12,6 @@ before_install:
 install:
   - "pip install xcffib"
   - "pip install ."
+  - "if [[ $TRAVIS_PYTHON_VERSION != pypy* ]]; then python cairocffi/ffi_build.py; fi"
```

Comment:
pip installing builds the file in `./build/lib/cairocffi/`, but importing the ffi object is done as `from ._ffi import ffi`, so it doesn't find it. The various fall backs let it run ok on its own, but when using xcffib, it needs the correctly built version.

The other option here would be to change the import to `from cairocffi._ffi import ffi`.